homework4/homework4.py

```diff
@@ -0,0 +1,123 @@
+from http import HTTPStatus
+
+from flask import Flask, jsonify
+
+import pandas as pd
+
+from webargs import fields, validate
+from webargs.flaskparser import abort, use_kwargs
+
+from database_handler import execute_query
+
+app = Flask(__name__)
+
+
+@app.errorhandler(HTTPStatus.UNPROCESSABLE_ENTITY)
+@app.errorhandler(HTTPStatus.BAD_REQUEST)
+def error_handler(error):
+    headers = error.data.get('headers', None)
+    messages = error.data.get('messages', ["Invalid request."])
+
+    if headers:
+        return jsonify(
+            {
+                'errors': messages
+            },
+            error.code,
+            headers
+        )
+    else:
+        return jsonify(
+            {
+                'errors': messages
+            },
+            error.code,
+        )
+
+
+@app.route('/price')
+@use_kwargs(
+    {
+        'country': fields.Str(
+            load_default=None,
+            validate=[validate.Regexp("[a-zA-Z]")]
+        ),
+    },
+    location="query"
+)
+def order_price(country):
+    sales = execute_query("SELECT invoices.BillingCountry, SUM(invoice_items.UnitPrice * invoice_items.Quantity) "
+                          "FROM invoices "
+                          "JOIN invoice_items "
+                          "ON invoice_items.InvoiceId = invoices.InvoiceId "
+                          "GROUP BY invoices.BillingCountry")
+    available_countries = [country[0] for country in sales]
```

Comment:
better to change the approach and create a query before executing it. 
So try to use only one query to the database. If country is incorrect and we don't have data for it show message that we don't have data for this country